src/components/Header/Header.js

```diff
@@ -181,29 +172,6 @@ class Header extends React.Component {
     }
   }
 
-  /**
-   * getHeaderHeight()
-   * returns the Height of the Header DOM
-   * element in pixels.
-   */
-  getHeaderHeight() {
-    const headerDOM = ReactDOM.findDOMNode(this.refs.nyplHeader);
-    return headerDOM.getBoundingClientRect().height;
-  }
-
-  /**
-   * setHeaderHeight()
-   * Updates the state headerHeight property
-   * only if headerHeight is not defined.
-   */
-  setHeaderHeight() {
-    if (!this.state.headerHeight) {
-      setTimeout(() => {
-        this.setState({ headerHeight: this.getHeaderHeight() });
-      }, 500);
-    }
-  }
-
   /**
    * getWindowVerticallScroll()
```

Comment:
This can probably be removed as well. This function was called as part of the stickyHeader logic to detect the vertical scroll position that would activate sticky header.

Reply:
Thanks. Removed.